docs/topics/proc_deploying-an-existing-wildfly-swarm-application-to-openshift.adoc

```diff
@@ -0,0 +1,60 @@
+
+[id='deploying-an-existing-wildfly-swarm-application-to-openshift]
+= Deploying an existing {WildFlySwarm} application to OpenShift
+
+You can easily deploy your existing application to OpenShift using the Fabric8 Maven plugin.
+
+.Prerequisites
+
+* A {WildFlySwarm}{ndash}based application
+
+.Procedure
+
+. Add the following profile to the `pom.xml` file in the root directory of your application:
++
+--
+[source,xml,options="nowrap",subs="attributes+"]
+----
+<profile>
+  <id>openshift</id>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>fabric8-maven-plugin</artifactId>
+        <version>{Fabric8MavenPluginVersion}</version>
+        <executions>
+          <execution>
+            <goals>
+              <goal>resource</goal>
+              <goal>build</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <generator>
+            <includes>
+              <include>wildfly-swarm</include>
+            </includes>
+            <excludes>
+              <exclude>webapp</exclude>
+            </excludes>
+          </generator>
+          <enricher>
+            <config>
+              <wildfly-swarm-health-check>
+                <path>/</path>
+              </wildfly-swarm-health-check>
+            </config>
+          </enricher>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</profile>
+----
+
+In this profile, the Fabric8 Maven plugin is invoked for building and deploying the application to OpenShift.
+--
+
+. Deploy the application to OpenShift according to instructions in xref:deploying-an-application-to-openshift[].
```

Comment:
Should this line in fact be an `include::topics/...something...`?

Reply:
@Ladicek it's true that including the module would also work here, but in this specific instance, we're cross-referencing a section that immediately precedes this content in the current guide structure. including the model again would make the section a bit redundant. An alternative would be to flip these 2 modules so that the deployment instructions follow rather than precede the section, but IMHO it is not a big issue. I won't hold this PR on it:)